DQM/DTMonitorModule/python/dtDQMOfflineSources_cff.py

```diff
@@ -50,3 +51,12 @@
                          dtEfficiencyMonitor +
                          dtTriggerEfficiencyMonitor +
                          dqmInfoDT)
+
+import EventFilter.DTRawToDigi.dturosunpacker_cfi
+_dturosunpacker = EventFilter.DTRawToDigi.dturosunpacker_cfi.dturosunpacker.clone()
+from Configuration.Eras.Modifier_run2_DT_2018_cff import run2_DT_2018
```

Comment:
era based modifications must be included in the python file that defines the module (and not propagated into the many other configurations that import it)

Reply:
DTDataIntegrityTask is defined in DQM/DTMonitorModule/python/dtDataIntegrityTask_cfi.py

Reply:
Ciao @perrotta yes, I fixed the first time you commented it, I'm confused now by your statements and those from @davidlange6 
Every module is modified only in the same file which defines it. I guess you and david were looking at an older version of the files, am I correct?
Thanks

Reply:
these are ok - I think I was confused last night.. sorry for the noise